Content.Shared/Fax/Components/FaxMachineComponent.cs

```diff
@@ -59,6 +59,13 @@ public sealed partial class FaxMachineComponent : Component
     [DataField]
     public bool ReceiveNukeCodes { get; set; } = false;
 
+    /// <summary>
+    /// Should fax connect to other maps faxes
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField]
+    public bool IsLongRange { get; set; } = false;
```

Comment:
`ViewVariables(VVAccess.ReadWrite)` is included in `DataField`
```suggestion
    [DataField]
    public bool IsLongRange { get; set; } = false;
```